spring-cloud-dataflow-server-mesos-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -52,16 +52,15 @@ You can pass in properties to set default values for memory and cpu resource req
 . Download and run the Spring Cloud Data Flow shell.
 +
 ```
-$ wget http://repo.spring.io/milestone/org/springframework/cloud/spring-cloud-dataflow-shell/1.0.0.M3/spring-cloud-dataflow-shell-1.0.0.M3.jar
+$ wget http://repo.spring.io/milestone/org/springframework/cloud/spring-cloud-dataflow-shell/1.0.0.M3/spring-cloud-dataflow-shell-1.0.0.RC1.jar
 
-$ java -jar spring-cloud-dataflow-shell-1.0.0.M3.jar
+$ java -jar spring-cloud-dataflow-shell-1.0.0.RC1.jar
 ```
 +
-. Register the Rabbit MQ version of the `time` and `log` app modules using the shell
+. By default, the application registry will be empty. If you would like to register all out-of-the-box stream applications built with the RabbitMQ binder in bulk, you can with the following command. For more details, review how to link:http://docs.spring.io/spring-cloud-dataflow/docs/{scdf-core-version}/reference/html/spring-cloud-dataflow-register-apps.html[register applications].
```

Comment:
Needs to be anchor else PDF and EPubs will take to a browser. i.e. <<streams.adoc#spring-cloud-dataflow-register-apps, register applications>>